pkgs/development/octave-modules/bim/default.nix

```diff
@@ -9,9 +9,11 @@ buildOctavePackage rec {
   pname = "bim";
   version = "1.1.6";
 
-  src = fetchurl {
-    url = "mirror://sourceforge/octave/${pname}-${version}.tar.gz";
-    sha256 = "sha256-pv64swrPlgopBlubpAlfoD9KJlOSgF9wdbgdHHTcr9c=";
+  src = fetchFromGitHub {
```

Comment:
How does this commit - changing some of the srcs' hosting website, affects the update script? Does the update script handle these hosting websites just as well?

Reply:
It actually makes things easier to switch to GitHub, as the script handles that more readily than fetchurl from SourceForge.

Reply:
According to my test with `statistics`, which switched to GitHub, indeed the update script seems fine.